tests/pipelines/dance_diffusion/test_dance_diffusion.py

```diff
@@ -31,6 +32,16 @@
 
 class DanceDiffusionPipelineFastTests(PipelineTesterMixin, unittest.TestCase):
     pipeline_class = DanceDiffusionPipeline
+    params = UNCONDITIONAL_AUDIO_GENERATION_PARAMS
+    required_optional_params = PipelineTesterMixin.required_optional_params - {
+        "callback",
+        "latents",
+        "callback_steps",
+        "output_type",
+        "eta",
+        "num_images_per_prompt",
```

Comment:
I'd group them into a meaningful variable and then use it instead of defining ad-hoc.

Reply:
yes this is a bit unreadable IMO, would be nicer to see directly what params are required

Reply:
Actually ok for me as is - I'd just move "eta" out of the "required" optional base params. ETA is too specific & we shouldn't incentivize adding it in the future